com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/JavaDebugDelegateCommandHandler.java

```diff
@@ -32,6 +34,18 @@
 
     public static String UPDATE_DEBUG_SETTINGS = "vscode.java.updateDebugSettings";
 
+    public static String START_REMOTE_DEBUG_SESSION = "vscode.java.startRemoteDebugSession";
```

Comment:
It's better not to mix the remote related code into the existing code? Move them into a separate class and package folder?